src/storage/accessors/FileDataAccessor.ts

```diff
@@ -117,12 +117,15 @@ export class FileDataAccessor implements DataAccessor {
   /**
    * Removes the corresponding file/folder (and metadata file).
    */
-  public async deleteResource(identifier: ResourceIdentifier): Promise<void> {
+  public async deleteResource(identifier: ResourceIdentifier): Promise<ResourceIdentifier[]> {
     const link = await this.resourceMapper.mapUrlToFilePath(identifier);
+    const metadataLink = await this.getMetadataLink(link.identifier);
     const stats = await this.getStats(link.filePath);
+    const modified: ResourceIdentifier[] = [ identifier ];
 
     try {
-      await fsPromises.unlink((await this.getMetadataLink(link.identifier)).filePath);
+      await fsPromises.unlink(metadataLink.filePath);
+      modified.push(metadataLink.identifier);
```

Comment:
I don't think we should do this. Currently there is no concept of metadata identifiers in the server. Metadata is always hidden in a RepresentationMetadata and this metadata identifier doesn't actually represent anything outside of the `FileDataAccessor`. Should we ever change so that we do expose metadata resources and allow accessing them directly this would make sense. But as of now, as far as the store is concerned, the `deleteResource` call will always only delete the one resource that is being requested.

The SparqlDataAccessor also has a metadata identifier btw, but there it makes even less sense since it starts with `meta:`.

Reply:
Agreed, will fix.